pkgs/tools/security/aide/default.nix

```
@@ -19,9 +19,9 @@ stdenv.mkDerivation rec {
     ];
 
   meta = with stdenv.lib; {
-    homepage = "http://aide.sourceforge.net/";
+    homepage = "https://aide.github.io/";
     description = "A file and directory integrity checker";
-    license = licenses.free;
+    license = licenses.gpl2;
```

```suggestion
    license = licenses.gpl2Plus;
```
https://github.com/aide/aide/blob/f2645b13adbef89648e5aa6689356265047ba7c8/COPYING#L296

---

sure, just learned about: https://github.com/NixOS/nixpkgs/blob/bc62fff2cfbcaf22fd18dac8a360d8e3a3410cfb/lib/licenses.nix#L803